Tests/AppTests/AnalyzerTests.swift

```diff
@@ -112,24 +113,26 @@ class AnalyzerTests: AppTestCase {
             
             // Git.revisionInfo (per ref - default branch & tags)
             // These return a string in the format `commit sha`-`timestamp (sec since 1970)`
-            // We simply use `fakesha` for the sha (it bears no meaning) and a range of seconds
+            // We simply use `sha` for the sha (it bears no meaning) and a range of seconds
             // since 1970.
             // It is important the tags aren't created at identical times for tags on the same
             // package, or else we will collect multiple recent releases (as there is no "latest")
-            if cmd.string == #"git log -n1 --format=format:"%H-%ct" "1.0.0""# { return "fakesha-0" }
-            if cmd.string == #"git log -n1 --format=format:"%H-%ct" "1.1.1""# { return "fakesha-1" }
-            if cmd.string == #"git log -n1 --format=format:"%H-%ct" "2.0.0""# { return "fakesha-0" }
-            if cmd.string == #"git log -n1 --format=format:"%H-%ct" "2.1.0""# { return "fakesha-1" }
-            if cmd.string == #"git log -n1 --format=format:"%H-%ct" "main""# { return "fakesha-2" }
-            
-            // Git.commitCount
-            if cmd.string == "git rev-list --count HEAD" { return "12" }
-            
-            // Git.firstCommitDate
-            if cmd.string == #"git log --max-parents=0 -n1 --format=format:"%ct""# { return "0" }
-            
-            // Git.lastCommitDate
-            if cmd.string == #"git log -n1 --format=format:"%ct""# { return "1" }
+            let tagToSha: [Reference: CommitHash] = [
+                .tag(1, 0, 0): "sha-0",
+                .tag(1, 1, 1): "sha-1",
+                .tag(2, 0, 0): "sha-2",
+                .tag(2, 1, 0): "sha-3",
+                .branch("main"): "sha-4",
+            ]
+            for (tag, sha) in tagToSha {
+                if cmd == .gitRevisionInfo(reference: tag, separator: "-") {
+                    return sha
+                }
+            }
+
+            if cmd == .gitCommitCount { return "12" }
+            if cmd == .gitFirstCommitDate { return "0" }
+            if cmd == .gitLastCommitDate { return "1" }
```

Comment:
This is an example how mocking gets easier and more readable: instead of matching on what we know the command is and then return a value we can just refer to the command by its name. No more guessing what `git log -n1 --format=format:"%H-%ct"` actually does.